### PR TITLE
harapan

### PR DESCRIPTION
**Pull request by mAlwiMhfd**

### CODE REVIEW

File: gitogether/1st-meeting/alwi.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+</head>
+<body>
+	<main>
+		<p>Nama: Muhammad Alwi Mahfud</p>
+		<p>Prodi: Teknik Informatika</p>
+		<p>Semester: 7</p>
+		<p>Harapan: STT NF dapat menjadi kampus IT terbaik</p>
+	</main>
+</body>
+</html>
```
Comment (orestotel) [inline gitogether/1st-meeting/alwi.html]: Everything is according to the provided template

File: gitogether/1st-meeting/alwi.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Harapan</title>
+</head>
```
Comment (orestotel) [inline gitogether/1st-meeting/alwi.html]: <head> tag is in place

File: gitogether/1st-meeting/alwi.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+<body>
+	<main>
+		<p>Nama: Muhammad Alwi Mahfud</p>
+		<p>Prodi: Teknik Informatika</p>
+		<p>Semester: 7</p>
+		<p>Harapan: STT NF dapat menjadi kampus IT terbaik</p>
+	</main>
```
Comment (orestotel) [inline gitogether/1st-meeting/alwi.html]: tag 'main' was used as intended

File: gitogether/1st-meeting/alwi.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
```
Comment (orestotel) [inline gitogether/1st-meeting/alwi.html]: right charset was included, which is UTF-8

File: gitogether/1st-meeting/alwi.html (hunk 2 above)
Comment (orestotel) [reply gitogether/1st-meeting/alwi.html]: The head tag, as it should be, is on the beginning of the document

**Comment by isfaaghyth** [comment]
please check all of the comments and fix it, @mAlwiMhfd

**Pull request merged**